src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -81,15 +84,24 @@ export class MongoStorageAdapter {
     });
   }
 
-  // Used for testing only right now.
-  allCollections() {
-    return this.connect().then(() => {
-      return this.database.collections();
-    }).then(collections => {
+  // Delete all data known to this adatper. Used for testing.
+  deleteAllSchemas() {
```

Comment:
the naming is somewhat misleading, not deleting all schemas from the schemas collection but actually deleting all objects from the database.